drivers/at/at.c

```diff
@@ -224,8 +268,7 @@ int at_send_cmd(at_dev_t *dev, const char *command, uint32_t timeout)
             return -1;
         }
 
-        if (at_expect_bytes(dev, CONFIG_AT_SEND_EOL AT_RECV_EOL_1 AT_RECV_EOL_2,
-                            timeout)) {
+        if (at_expect_bytes(dev, CONFIG_AT_SEND_EOL, timeout)) {
```

Comment:
BTW. I created https://github.com/RIOT-OS/RIOT/pull/20281 because it allows me to see exactly what the modem is returning for an AT command. For example:
```
2024-01-24 22:26:30,373 # send 1 AT+COPS?
2024-01-24 22:26:30,374 # UART_DEV(1) TX: AT+COPS?
> 2024-01-24 22:26:30,401 # Success: UART_DEV(1) RX: [AT+COPS?\r\r\n]
2024-01-24 22:26:30,403 # Success: UART_DEV(1) RX: [+COPS: 0\r\n]
2024-01-24 22:26:30,404 # Success: UART_DEV(1) RX: [\r\n]
2024-01-24 22:26:30,405 # Success: UART_DEV(1) RX: [OK\r\n]
```